### PR TITLE
Added logic for handling internal server Error 

### PR DESCRIPTION
**Pull request by maxburbeliuk**

### CODE REVIEW

File: src/navigators/AuthNavigator/authRoutes.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 export const AUTH_ROUTES = [
   { key: LOGIN, path: LOGIN, element: <Login /> },
   {
     key: NOT_FOUND,
     path: '*',
     element: <Navigate to={`services${NOT_FOUND}`} />
+  },
+  {
+    key: INTERNAL_SERVER_ERROR,
+    path: '*',
```
Comment (okrdima) [inline src/navigators/AuthNavigator/authRoutes.js:18]: What is actually difference between real path and *

File: src/navigators/ServicesNavigator/servicesRoutes.js [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
 export const NOTFOUND_ROUTES = [
   { key: NOT_FOUND, path: NOT_FOUND, element: <NotFound /> },
-  { key: '404', path: '*', element: <Navigate to={NOT_FOUND} /> }
+  { key: '404', path: '*', element: <Navigate to={NOT_FOUND} /> },
+  {
+    key: INTERNAL_SERVER_ERROR,
+    path: INTERNAL_SERVER_ERROR,
+    element: <InternalServerError />
+  },
+  {
```
Comment (okrdima) [inline src/navigators/ServicesNavigator/servicesRoutes.js:16]: Why you need to  duplicate this with prev item?

File: src/navigators/AuthNavigator/authRoutes.js [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
 const { NOT_FOUND } = SERVICES_PATHS
+const { INTERNAL_SERVER_ERROR } = ISE_PATHS
 export const AUTH_ROUTES = [
   { key: LOGIN, path: LOGIN, element: <Login /> },
   {
     key: NOT_FOUND,
     path: '*',
     element: <Navigate to={`services${NOT_FOUND}`} />
+  },
+  {
```
Comment (okrdima) [inline src/navigators/AuthNavigator/authRoutes.js:16]: you add it for auth route but what about app routes?

File: src/navigators/AuthNavigator/authRoutes.js (hunk 1 above)
Comment (maxburbeliuk) [reply src/navigators/AuthNavigator/authRoutes.js:18]: * In the react router, this is a universal route that corresponds to any path of the URL address
A "route path" in React Router is a string that represents a URL pattern. For example, `/home`, `/about`, or `/products/:id`

File: src/navigators/ServicesNavigator/servicesRoutes.js (hunk 2 above)
Comment (maxburbeliuk) [reply src/navigators/ServicesNavigator/servicesRoutes.js:16]: Fallback Routes - Each duplicate route may serve a different error handling strategy or approach

File: src/navigators/AuthNavigator/authRoutes.js (hunk 3 above)
Comment (maxburbeliuk) [reply src/navigators/AuthNavigator/authRoutes.js:16]: My bad, I'll fix it now

**Pull request closed without merging**